lib/Manager.php

```diff
@@ -199,6 +191,24 @@ public function createParticipantObject(Room $room, array $row): Participant {
 		);
 	}
 
+	public function createCommentObject(array $row): ?IComment {
+		return $this->commentsManager->getCommentFromData([
+			'id'              => $row['comment_id'],
+			'parent_id'        => $row['comment_parent_id'],
+			'topmost_parent_id' => $row['comment_topmost_parent_id'],
+			'children_count'   => $row['comment_children_count'],
+			'message'         => $row['comment_message'],
+			'verb'            => $row['comment_verb'],
+			'actor_type'       => $row['comment_actor_type'],
+			'actor_id'         => $row['comment_actor_id'],
+			'object_type'      => $row['comment_object_type'],
+			'object_id'        => $row['comment_object_id'],
+//			'reference_id'        => $row['comment_reference_id'] ?? null,
```

Comment:
`php-cs` wants the comment to be indented. And I would appreciate an additional comment explaining why the `reference_id` is commented :-)

Reply:
My brain was overloaded, and I forgot to find out how we detect whether the column has been creates already.
`if ($this->config->getAppValue('spreed', 'has_reference_id', 'no') === 'yes')` was it, copied now from capabilities